lib/assertions.ts

```diff
@@ -836,7 +836,7 @@ export default class DOMAssertions {
       return [];
 
     } else if (typeof this.target === 'string') {
-      return Array.from(this.rootElement.querySelectorAll(this.target));
+      return [].slice.call(this.rootElement.querySelectorAll(this.target));
```

Comment:
Doesn't make this code more readable IMO. Maybe adding an `arrayFrom()` utility function to hide that complexity?

Reply:
Personally, I think a regression in readability is worth it if it fixes the experience for IE11. And I also wouldn't like to add more code than was necessary to get the job done, adding a utility function would make this PR much larger 🙈

Reply:
Maybe `Array.prototype.slice.call()` would be slightly better performing, as it would not create a useless empty array that would need to be garbage collected? But probably neglectable...

A quick bug fix release would be appreciated though! 😁